maxim/apis/maxim_apis.py

```
@@ -689,45 +695,209 @@ def get_folders(self) -> List[Folder]:
+        
+        Args:
+            attachment: The URL attachment to process
+            
+        Returns:
+            tuple: (file_data, mime_type, size)
+        """
+        try:
+            with self.connection_pool.get_client() as client:
+                # Validate URL
+                if not attachment.url or not attachment.url.startswith(('http://', 'https://')):
+                    raise ValueError(f"Invalid URL: {attachment.url}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)